actions/mongodb_explain_action_test.go

```diff
@@ -0,0 +1,82 @@
+package actions
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/percona/percona-toolkit/src/go/mongolib/proto"
+	"github.com/percona/pmm/api/agentpb"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/percona/pmm-agent/utils/tests"
+)
+
+func TestMongoDBExplain(t *testing.T) {
+	database := "test"
+	collection := "test_col"
+	id := "abcd1234"
+	ctx := context.TODO()
+
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(tests.MongoDBDSN()))
+	if err != nil {
+		t.Fatalf("Cannot connect to MongoDB: %s", err)
+	}
+
+	if err := prepareData(ctx, client, database, collection); err != nil {
+		t.Fatalf("Cannot prepare MongoDB testing data: %s", err)
+	}
+
+	eq := proto.ExampleQuery{
+		Ns: "test.coll",
+		Op: "query",
+		Query: proto.BsonD{
+			{
+				Key: "k",
+				Value: proto.BsonD{
+					{
+						Key:   "$lte",
+						Value: int32(1),
+					},
+				},
+			},
+		},
+		Command:            nil,
+		OriginatingCommand: nil,
+		UpdateObj:          nil,
+	}
+	buf, _ := bson.MarshalExtJSON(eq, true, true)
+
+	params := &agentpb.StartActionRequest_MongoDBExplainParams{
+		Dsn:      tests.MongoDBDSN(),
+		Database: database,
+		Query:    string(buf),
+	}
+
+	ex := NewMongoDBExplain(id, params)
+	_, err = ex.Run(ctx)
```

Comment:
shouldn't we test output of explain action?

Reply:
I didn't thought it was necessary because the explain package itself has a LOT of tests with different queries and different MongoDB versions.
In my opinion, if the explain works as expected should be tested in the explain package.
Of course, we can do it here too.

Reply:
So maybe we need one test to be sure that we are getting some output?